Source/Web View/WebViewController.swift

```diff
@@ -142,7 +142,20 @@ public class WebViewController: UIViewController {
     
     /// Set `false` to disable error message UI.
     public var showErrorDisplay = true
-    
+
+    public var userAgent: String?
+
+    public lazy var urlSession: NSURLSession = {
```

Comment:
do you think the session needs to be public? I think we should keep the web view's request loading system internal since the details don't matter to external users and I don't ever see us wanting to allow this to be customized. I think it is better to make this completely private or read-only public